lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -0,0 +1,829 @@
+{
+   "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
+   "contentVersion": "1.0.0.0",
+   "parameters": {
+      "msiUri": {
+         "type": "string",
+         "defaultValue": "https://lansalpcmsdn.blob.core.windows.net/app/test/AWAMAPP_v14.1.2_en-us.msi",
```

Comment:
The MSI is unique for each deployment. It is created using our Visual LANSA IDE SKU. I think its appropriate for it to be in a custom storage account. The deployment of the MSI is the whole reason for the template in the first place. The template exists to host the application embodied by the MSI.